playground/vitest.config.e2e.ts

```diff
@@ -1,7 +1,7 @@
 import { resolve } from 'node:path'
 import { defineConfig } from 'vitest/config'
 
-const timeout = process.env.CI ? 20_000 : 5_000
+const timeout = process.env.PWDEBUG ? Infinity : process.env.CI ? 20_000 : 5_000
```

Comment:
This is the same trick I added in Vite 6 PR. It allows `PWDEBUG=1 pnpm test-serve` to avoid timeout and use playwright debugger/recorder UI more freely.